### ISSUE TITLE
rhythmbox icon is not recognizable

### ISSUE THREAD

**Issue by matthewpaulthomas** [snwh/suru-icon-theme#90]
yaru-theme-* 18.10.4, Ubuntu Cosmic

If you’ve used Rhythmbox frequently before, perhaps in previous versions of Ubuntu, you might remember the unusual colour scheme of the previous icon:

![193px-rhythmbox_logo svg](https://user-images.githubusercontent.com/19801137/45930476-c3ee4600-bf58-11e8-81f7-c37416d45723.png)

But if you haven’t, or don’t, then the music-player icon (aliased to Rhythmbox in #78) looks like nothing so much as a big yellow eye.

![icon for rhythmbox](https://user-images.githubusercontent.com/19801137/45930454-6fe36180-bf58-11e8-9e3f-68d0c33429e7.png)

Apart from reusing Rhythmbox’s old colors, there’s practically nothing to suggest that the icon is a speaker:
- It’s an unusual shape for a speaker — and since all other app icons are the same shape, it’s not obviously a physical object at all. There is no visibility of the top or side of the speaker, no shadow on the surface it’s resting on, and no reflection of the surface on the speaker itself, while the previous icon had all three. (If those are all unavoidable limitations of the icon style, then the other design aspects need to work that much harder.)
- It’s an unusual set of colors: [speakers with white bodies seldom have yellow cones](https://duckduckgo.com/?q=white+loudspeaker&t=canonical&iax=images&ia=images), and [speakers with yellow cones seldom have white bodies](https://duckduckgo.com/?q=speaker+%22yellow+cone%22&t=canonical&iax=images&iar=images&ia=images).
- Now that the dustcap is black rather than silver, its pointed shape is barely visible.
- There are no visible screws, no nameplate, and no cable.
- There are no sound waves, like the previous icon had, or music notes.

[[Copied from yaru#859](https://github.com/ubuntu/yaru/issues/859).]

**Comment by snwh** [comment]
I know and I pretty much agree with all your points. I don't feel that the Suru app-icon style is conducive for making "in-style" icons for every application and my intention was never to have every app icon have a Suru icon, just the "generic" ones.

Those listed reason were why I hadn't included a Rhythmbox icon in the set initially and had just linked to the Music icon. I included one as a concession to folks who wanted a dedicated Rhythmbox icon in the set. (I advocated the dropping of Rhythmbox for GNOME Music but that's a different issue).

So, my solution would be revert and drop the Suru Rhythmbox icon.

**Comment by matthewpaulthomas** [comment]
Ah, sorry, I had misunderstood @Feichtmeier as saying that this _was_ the music-app icon, but now I see [that’s very different](https://github.com/snwh/suru-icon-theme/blob/master/src/fullcolor/apps/music-app.svg).

> I don't feel that the Suru app-icon style is conducive for making "in-style" icons for every application and my intention was never to have every app icon have a Suru icon, just the "generic" ones.

Intriguing. [Yaru is on a quest for Suru-style icons for all _default_ apps](https://github.com/ubuntu/yaru/issues/821). I guess you mean something quite different by “generic” apps, and I’d love to know what that is. In other words, what should a user be able to infer about an app, from its icon being Suru-style or not?

**Comment by Feichtmeier** [comment]
I think he means apps that have a generic name and branding like

gnome music or
elementary os music
They both are generic music apps
Whereas rhythmbox is supposed to be a little bit more branded
This could come out of a time where all gnome apps had some little spicy name like totem or Nautilus
And now they want to go to the generic names gnome files and gnome videos
If Suru drops this icon we will provide an icon from our side since this old school icon is very much not what we want for default apps. Maybe we are all lucky and 19.4 will ship with gnome music but my hopes are low for that

**Comment by Feichtmeier** [comment]
Then close this one and we continue this downstream

**Comment by snwh** [comment]
@Feichtmeier I mean those in the FreeDesktop icon specification, plus a few others (namely in Suru what are `*-app` icons.

**Comment by snwh** [comment]
@matthewpaulthomas the [FreeDesktop icon spec](https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#apps) does not have very many generic/standard application icons. Humanity, for example, is a complete FreeDesktop icon set (plus some extras) and it shipped very few application icons, i.e. is no Humanity Rhythmbox icon, but I've generally lumped in the GNOME apps as "generic" so there are at least a few Suru app icons.

What I don't think it's appropriate for a distribution (especially one as major as Ubuntu) to ship a theme which overrides *all* default applications' branding, whether that be Firefox or Transmission, etc.

**Comment by matthewpaulthomas** [comment]
I agree it’s a dubious goal to aim for rounded-rectangle icons of all default apps, though maybe for a different reason. Even if it was achievable, if (for example) your IT department replaced Firefox with Chrome or Opera, its icon would stick out weirdly.

Whatever the reason, though, it still leaves the question of where to draw the line between things that have rounded-rectangle icons and things that don’t. That is, what people are supposed to infer from the shape. “The rounded ones are in the FreeDesktop icon spec” would not mean anything to people who don’t, and will never need to, know what FreeDesktop is. And “the rounded ones are generic apps” would also mean nothing: Files is still Nautilus under the hood, and is deemed “generic”, while Rhythmbox is still Rhythmbox under the hood, but somehow is not. Is it just because another app with a more-generic name (“Music”) now exists? If someone renamed Rhythmbox to “Audio”, would it become the generic one? :wink:

I know of two cases where icon style has communicated something about _kinds of_ app (not just vendor, but genre), both of them in macOS. In the 1980s/1990s, document-based app icons often featured a 45°-rotated sheet of paper with a tool over it (e.g. [TeachText](https://en.wikipedia.org/wiki/TeachText), [MacWrite](https://en.wikipedia.org/wiki/MacWrite), [FreeHand](https://www.macintoshrepository.org/694-macromedia-freehand-5-5)). And in Mac OS X, [Utilities icons are usually monochrome](https://512pixels.net/projects/aqua-screenshot-library/macos-10-14-mojave/#jp-carousel-17019), compared with non-utility apps that are usually brightly colored. For the long term, maybe it’s worth considering distinct shapes for things like “this app is for configuring the system” or “this app is for working with audio/video”. Besides being more expressive, the variety would also make non-Suru app icons stick out less.

**Comment by snwh** [comment]
@matthewpaulthomas right. You're convincing me that the best course is to not use the Suru style/rounded-rectangle at all for any application icons, however it's too late refactor the Suru style/guidelines. 😉

> where to draw the line between things that have rounded-rectangle icons and things that don’t

It's pretty impossible to draw a satisfactory line and the one I did is fairly arbitrary so I have to end up being dictatorial because folks will say "why doesn't X have an icon?".

I do/did like the macOS approach where style delineates function (they somewhat do it with the 9&deg; tilted icons these days but not consistently).

FWIW, I'd totally rethink aspects of this set, but it was meant to be one of those free-time-free-software projects one has, and this quasi-official status is pulling out all the flaws in the set that I wish I had the time to fix/change (but I don't).

**Comment by Ads20000** [comment]
> this quasi-official status is pulling out all the flaws in the set that I wish I had the time to fix/change (but I don't)

Do you have the time to accept possible PRs from the Yaru team attempting to fix those flaws? Or perhaps they need commit access?

**Comment by matthewpaulthomas** [comment]
Submit it and see! If not, it can always be resubmitted downstream.

If you want to keep the shape, but a speaker can’t be recognizable enough in this shape, perhaps a top-down view of a turntable? That can have many more details that could contribute to recognizability: cartridge and tone arm, knobs, pitch slider, cue lever, illuminator, and a record with a rotated label.

**Comment by snwh** [comment]
@Ads20000 I have always been open to (and encouraged) PRs the whole time. 😉

**Comment by snwh** [comment]
@matthewpaulthomas what's a turntable? 👶 I'd abstract a turntable, if anything, to its more recognizable aspects-the LP and the arm–other details of the turntable would be pretty superfluous in an icon form.
